packages/kbn-search-types/src/types.ts

```diff
@@ -114,6 +114,11 @@ export interface ISearchOptions {
    * To pass an abort signal, use {@link ISearchOptions.abortSignal}
    */
   transport?: Omit<TransportRequestOptions, 'signal'>;
+
+  /**
+   * When set es results are streamed back to the caller without any parsing of the content.
+   */
+  stream?: boolean;
```

Comment:
Any reason not to just keep the name the same as the ES option? (`asStream`)

Reply:
not really, I didn't see a reason for it as we don't generally align our prop names with es.